### DISCUSSION TITLE
Invalid ROM name

### DISCUSSION THREAD

**Discussion by stevanwilson** [sanni/cartreader#714] [Need help]
Hi All, I built the Cart Reader v3. I recently picked up a copy of Earthbound and I'm trying to save the ROM. The name of the file is \AHBOUND. This is failing to write the ROM with the following Message: 

"Saving to SNES/ROM/\AHBOUND/52/... 
Can't create file on SD"

I'm guessing this is an issue with the name of the File: \AHBOUND not being a valid name of a directory of file. Is there a way around this?

**Comment by PsychoFox11** [comment]
I don't have a copy of Earthbound on hand, but I've gotten bad titles when a cart was just a tad dirtier. I'd think if it did this with every Earthbound, a lot more people would be complaining. I'd guess if you did get a dump the CRC32 would say not found. I've found with my Sanni, that the games need to be extra extra clean. Pickier than the consoles. The plus is, all my games are becoming really clean now! However, 90% of my dumping time has been cleaning haha. It's nice and meditative though!
I'll let someone else comment if this is like a weird variation, but I'd guess very slightly dirty cart.

**Comment by Chomemel** [comment]
Yeah I had that once with UPER MARIO RPG and BOCKMAN X 2 ^^. After a good cleaning, I had the names right.

**Comment by stevanwilson** [comment]
Thanks for the responses! I tried cleaning the contacts and retested a few times and it finally worked!
